cloudformation-template.json

```diff
@@ -170,8 +167,8 @@
 							"Headers" : [],
 							"QueryString" : "true"
 						},
-						"MinTTL" : "0",
-						"DefaultTTL":"0",
+						"MinTTL" : "63115200",
+						"DefaultTTL":"63115200",
```

Comment:
Yeah so now we force the TTL rather than setting it in the lambda-server. Less config / logic in the lambda function which I think is better.